web-www/pages/index.tsx

```diff
@@ -37,9 +38,26 @@ export default function Home({
   const statsData = allHomePageWeb.homeWebStatsSection;
   const valuesData = allHomePageWeb.valuesSection;
   const ledgerDescription = allHomePageWeb.ledgerDescription;
+  const GA_MEASUREMENT_ID = process.env.NEXT_PUBLIC_GA_MEASUREMENT_ID;
 
   return (
     <>
+      {GA_MEASUREMENT_ID && (
+        <>
+          <Script
+            src={`https://www.googletagmanager.com/gtag/js?id=${GA_MEASUREMENT_ID}`}
+          />
+          <Script id="google-analytics">
+            {`
+          window.dataLayer = window.dataLayer || [];
+          function gtag(){dataLayer.push(arguments);}
+          gtag('js', new Date());
+
+          gtag('config', '${GA_MEASUREMENT_ID}');
+        `}
```

Comment:
should we reformat this correctly?

Reply:
somehow the linter wasn't doing anything for me on this block. i'll try to fix that too

Reply:
prettier doesn't seem to do anything for this block. i think it's because they have to assume whitespaces are meaningful in a multiline template string.... but in any case i think i did improve the formatting and i did add a fix for the web-www package format-and-format command